### PR TITLE
add event listeners to switch views

### PR DESCRIPTION
**Pull request by Atos20**
Hey Greyson I think we are doing a great job, keep it up.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -4,9 +4,10 @@ var homeButton = document.querySelector('.home-button');
 var randomizeButton = document.querySelector('.random-cover-button');
 var saveCoverButton = document.querySelector('.save-cover-button');
 var viewSavedButton = document.querySelector('.view-saved-button');
-var makeNewButton = document.querySelector('.make-new-button')
+var userCoverButton = document.querySelector('.make-new-button')
```
Comment (GreysonElkins) [inline src/main.js:7]: I think this is a good change, and that we should consider renaming our button variables and functions with a clearer naming convention. (But not for this iteration.)

File: src/main.js [hunk 2]
```diff
@@ -15,8 +16,36 @@ var currentCover ;
...
+randomizeButton.classList.add('hidden');
+saveCoverButton.classList.add('hidden');
+homeButton.classList.remove('hidden');
+}
+
+function viewSavedPage() {
+  mainCover.classList.add('hidden');
+  savedCoversPage.classList.remove('hidden');
+  randomizeButton.classList.add('hidden');
+  // viewSavedButton.classList.add('hidden');
```
Comment (GreysonElkins) [inline src/main.js]: Let's delete this line while learning how to resolve a comment which prevents the PR from being merged.

**Review by GreysonElkins — CHANGES_REQUESTED** [review]
Do you mind if we try out the request changes function together?

**Comment by GreysonElkins** [comment]
Hey Orlando,

I think we're doing great too, and we should high five over beers very soon.

One small change for the sake of getting used to GitHub.

File: src/main.js (hunk 1 above)
Comment (Atos20) [reply src/main.js:7]: cool let's do that next time we meet and we can go over it.

**Comment by GreysonElkins** [comment]
I see you made that change! Let's round up with Scott at some point to make sure we're resolving the "Changes requested" properly when working in github <3

**Pull request merged**